### PR TITLE
[Components with more than 1 service] LPS-181737 Refactor PortletToolbarContributor components from Lima modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-181737

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:lima

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:lima

**Comment by Kvale1733** [comment]
ci:test:startup

File: modules/apps/bookmarks/bookmarks-web/src/main/java/com/liferay/bookmarks/web/internal/portlet/action/ViewMVCRenderCommand.java [hunk 1]
```diff
@@ -74,8 +74,9 @@ public String render(
 		return "/bookmarks/view.jsp";
 	}
 
-	@Reference
-	private BookmarksPortletToolbarContributor
-		_bookmarksPortletToolbarContributor;
+	@Reference(
+		target = "(component.name=com.liferay.bookmarks.web.internal.portlet.toolbar.contributor.BookmarksPortletToolbarContributor)"
+	)
+	private PortletToolbarContributor _bookmarksPortletToolbarContributor;
```
Comment (rafaprax) [inline modules/apps/bookmarks/bookmarks-web/src/main/java/com/liferay/bookmarks/web/internal/portlet/action/ViewMVCRenderCommand.java:80]: Hey @MrJohn1911, We can remove the cast from [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/bookmarks/bookmarks-web/src/main/java/com/liferay/bookmarks/web/internal/display/context/BookmarksManagementToolbarDisplayContext.java#L112) safely. Please do that. Thx

File: modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/portlet/action/ViewPagesMVCRenderCommand.java [hunk 2]
```diff
@@ -79,7 +79,9 @@ private String _getPortletId(RenderRequest renderRequest) {
 	@Reference
 	private WikiEngineRenderer _wikiEngineRenderer;
 
-	@Reference
-	private WikiPortletToolbarContributor _wikiPortletToolbarContributor;
+	@Reference(
+		target = "(component.name=com.liferay.wiki.web.internal.portlet.toolbar.item.WikiPortletToolbarContributor)"
+	)
+	private PortletToolbarContributor _wikiPortletToolbarContributor;
```
Comment (rafaprax) [inline modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/portlet/action/ViewPagesMVCRenderCommand.java:85]: Hey @MrJohn1911 , please apply the same approach [here]( https://github.com/liferay/liferay-portal/blob/master/modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/display/context/WikiPagesManagementToolbarDisplayContext.java#L161). Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by tinatian** [comment]
ci:forward

**Pull request closed without merging**